wake_word.py

```diff
@@ -3,24 +3,31 @@
 import openai
 from dotenv import load_dotenv
 import os
+import tiktoken  # Import the tiktoken library for token counting
 
 # Load environment variables from the .env file
 load_dotenv()
 
 # Get the API key from the environment variable
 openai.api_key = os.getenv('OPENAI_API_KEY')
 
+# Tokenizer setup for total tokens used calculation
+tokenizer = tiktoken.get_encoding("gpt2")
```

Comment:
https://github.com/openai/tiktoken/blob/main/tiktoken_ext/openai_public.py
because gpt 40-mini is not working not even gpt 3.5 turbo .it says that the tokenizer can not recognize that .i have added a link from where i have added a gpt 2

Reply:
yes i used that first but gpt 2 was also working so i added that, what is your suggestion ?

Reply:
alright?i was not aware of the differences they create .**cl100k_base** is more advanced when you are using gpt 4 models ,i will change it.